deps/build_p7zip.v16.2.0.jl

```diff
@@ -0,0 +1,51 @@
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
+
+# Parse some basic command-line arguments
+const verbose = "--verbose" in ARGS
+const prefix = Prefix(get([a for a in ARGS if a != "--verbose"], 1, joinpath(@__DIR__, "usr")))
+products = [
+    ExecutableProduct(prefix, "7z", :p7zip_exe),
+]
+
+# Download binaries from hosted location
+bin_prefix = "https://github.com/JuliaBinaryWrappers/p7zip_jll.jl/releases/download/p7zip-v16.2.0+4"
+
+# Listing of files generated by BinaryBuilder:
+download_info = Dict(
+    # MacOS(:aarch64) => ("$bin_prefix/p7zip.v16.2.0.aarch64-apple-darwin.tar.gz", "4394e087868097772a0f9a13c9069522192dc9488d36fbf37f37b8328ad9b031"),
+    Linux(:aarch64, libc=:glibc) => ("$bin_prefix/p7zip.v16.2.0.aarch64-linux-gnu.tar.gz", "7c6134329060d35ba4ec301ef7b2d9a906928f428ca9c683a0e4630ee66d54f8"),
+    Linux(:aarch64, libc=:musl) => ("$bin_prefix/p7zip.v16.2.0.aarch64-linux-musl.tar.gz", "9d20b930c0fa8e833103dae2c7dae82072f2a20f2a669945a08a8099e519138f"),
+    # Linux(:armv6l, libc=:glibc, call_abi=:eabihf) => ("$bin_prefix/p7zip.v16.2.0.armv6l-linux-gnueabihf.tar.gz", "e18ddbda3e1327806aab32ffb7732355e297995daa3e94835ed299ef6d7bbac6"),
+    # Linux(:armv6l, libc=:musl, call_abi=:eabihf) => ("$bin_prefix/p7zip.v16.2.0.armv6l-linux-musleabihf.tar.gz", "6423ead668273a62908cb076ec884f7793b30380fb7ca9c1c97dbffdfafc0172"),
```

Comment:
```suggestion
```

Reply:
I'd prefer leaving these commented out as if you regenerate this file you know that these were explicitly disabled and were not just missing

Reply:
Fair! I'm unable to resolve this thread.